app/controllers/manifests_controller.rb

```diff
@@ -1,4 +1,8 @@
 class ManifestsController < ApplicationController
   def new
   end
+
+  def index
+    @manifests = Manifest.all
```

Comment:
This is fine for now. I have a pending PR that will change it to use `authorized_search({public: true})`